code/modules/mob/dead/new_player/sprite_accessories.dm

```diff
@@ -1511,6 +1511,18 @@
 	name = "Thigh-high (Bee)"
 	icon_state = "bee_thigh"
 
+/datum/sprite_accessory/socks/codersocks_pink
+	name = "Coder Socks (Pink)"
+	icon_state = "codersocks_pink"
+
+/datum/sprite_accessory/socks/codersocks_blue
+	name = "Coder Socks (Blue)"
+	icon_state = "codersocks_blue"
+
+/datum/sprite_accessory/socks/codersocks_trans
+	name = "Coder Socks (Trans)"
```

Comment:
Ah yes, Trans, my favourite color

Reply:
Do you have a better word for pink blue white blue pink

Reply:
what about pink-blue-white-blue-pink?

Reply:
> 
> 
> Do you have a better word for pink blue white blue pink

pastel stripes